### PR TITLE
syslogd: fix denials from systemd-journald

### PR DESCRIPTION
**Pull request by cgzones**

### CODE REVIEW

File: policy/modules/system/logging.te [hunk 1]
```diff
@@ -504,6 +504,12 @@ ifdef(`init_systemd',`
 	allow syslogd_t self:capability { chown setuid setgid };
 	allow syslogd_t self:netlink_audit_socket { getattr getopt read setopt write };
 
+	# create /var/run/log
+	files_pid_filetrans(syslogd_t, var_log_t, dir, "log")
```
Comment (pebenito) [inline policy/modules/system/logging.te]: I don't have a `/var/run/log` on my systemd system, what is this from?

File: policy/modules/system/logging.te [hunk 2]
```diff
@@ -512,6 +518,8 @@ ifdef(`init_systemd',`
 	init_daemon_pid_file(syslogd_var_run_t, dir, "syslogd")
 	init_dgram_send(syslogd_t)
 
+	fs_search_cgroup_dirs(syslogd_t)
```
Comment (pebenito) [inline policy/modules/system/logging.te]: This goes at line 511.

**Comment by cgzones** [comment]
/run/log seems to be the store for the current journald logs
```
root@debianse:~# ll /var/run/log/journal/fe1d4a4d0ac7441e817d011e70da2471/
total 29M
drwxr-s---+ 2 root systemd-journal system_u:object_r:var_log_t:s0  140 Feb 16 09:09 .
drwxr-sr-x. 3 root systemd-journal system_u:object_r:var_log_t:s0   60 Nov  3 18:16 ..
-rw-r-----+ 1 root systemd-journal system_u:object_r:var_log_t:s0 5.8M Feb 16 15:27 system.journal
-rw-r-----+ 1 root systemd-journal system_u:object_r:var_log_t:s0 5.8M Feb 14 18:31 system@d6240401ac244193890964694d3d5feb-0000000000000001-00054068b8b1dd69.journal
-rw-r-----+ 1 root systemd-journal system_u:object_r:var_log_t:s0 5.8M Feb 15 07:01 system@d6240401ac244193890964694d3d5feb-00000000000000d5-00054880ef846bed.journal
-rw-r-----+ 1 root systemd-journal system_u:object_r:var_log_t:s0 5.8M Feb 15 20:12 system@d6240401ac244193890964694d3d5feb-0000000000000982-0005488b67a0dcc2.journal
-rw-r-----+ 1 root systemd-journal system_u:object_r:var_log_t:s0 5.8M Feb 16 09:09 system@d6240401ac244193890964694d3d5feb-00000000000012f2-00054896776873ee.journal
```

https://www.freedesktop.org/software/systemd/man/systemd-journald.service.html:
> By default, the journal stores log data in /run/log/journal/.

**Comment by cgzones** [comment]
really??
why not merging this
> I don't have a /var/run/log on my systemd system, what is this from?

and then apply the patches from Russel without a doubt
https://github.com/TresysTechnology/refpolicy/commit/2087bde93412d9bbbb92fd8316b5e8176451d836#diff-1d64ea97de0be86e573c655de560bbc1R410

**Comment by pebenito** [comment]
I'm sorry if you're offended, but it definitely wasn't without doubt or questions. In this case, I saw that his changes had a different label for the journal, and decided that was a good choice.

**Comment by doverride** [comment]
I suspect that you probably do not have persistent logging enabled with systemd-journald. If you do then i think you should be seeing /run/log

**Comment by doverride** [comment]
either that or maybe you're still using the old syslog with systemd

**Pull request closed without merging**